graphql_compiler/schema_transformation/utils.py

```diff
@@ -33,7 +34,33 @@ class InvalidTypeNameError(SchemaTransformError):
 
 
 class SchemaNameConflictError(SchemaTransformError):
-    """Raised when renaming types or fields cause name conflicts."""
+    """Raised when renaming or merging types or fields cause name conflicts."""
+
+
+_alphanumeric_and_underscore = frozenset(string.ascii_letters + string.digits + '_')
+
+
+def check_schema_identifier_is_valid(identifier):
+    """Check if input is a valid identifier, made of alphanumeric and underscore characters.
+
+    Args:
+        identifier: str
```

Comment:
Might be nice to give the reader some intuition about what `identifier` we're talking about. The errors all mention `schema identifier` but the docstring doesn't really say what the identifier is supposed to be identifying.

In general, I'm not a fan of just defining the type of the argument without attempting to provide some intuition about where that value might be coming from / what it might be used for.